boards/arm/rddrone_fmuk66/Kconfig.defconfig

```diff
@@ -23,6 +23,9 @@ config MCG_FCRDIV
 config I2C
 	default y
 
+config SPI
+	default y
+
```

Comment:
This change shouldn't be necessary now that sensor drivers select SPI when they need it

Reply:
Acknowledged